garrysmod/lua/vgui/prop_combo.lua

```diff
@@ -15,8 +15,15 @@ function PANEL:Setup( vars )
 	combo:DockMargin( 0, 1, 2, 2 )
 	combo:SetValue( vars.text or "Select..." )
 
+	local hasIcons, icon = istable( vars.icons )
 	for id, thing in pairs( vars.values or {} ) do
-		combo:AddChoice( id, thing )
+		if(hasIcons) then
```

Comment:
Spacing is required here

Reply:
Fixed in fbf1e00debbf39c8e81097dc351453f2ff98d30d

Reply:
You did not fix it, there is still no space between if and the parenthesis.

Reply:
Still missing the space on the other side of the if..